server/models/Profile.js

```diff
@@ -35,11 +35,6 @@ const profileSchema = new mongoose.Schema({
     type: String,
     default: "",
   },
-  userId: {
-    type: mongoose.Schema.Types.ObjectId,
-    required: true,
-    ref: "User",
-  },
```

Comment:
I'm a bit confused about deleting this as it seems not having this link would complicate things🤔 
When we list sitter profiles to an owner, arguably we should do it via the `Profile` model since it includes all the related information about a request (e.g. availability, address, etc.). Therefore we will return a bunch of fitting profile ids to the frontend, and things won't work if all they have profile ids when trying to create a request or send a message... We will need a way to associate profile ids to their accounts/user ids in that case 👍🏼 

The alternative would be to fetch User documents and populate their profiles, then apply the search/filter based on the `profile` field and return the user along with the profile... Seems a lot of overhead and avoidable complexity, what do you think?

Reply:
I agree with you. Let's have it.

Reply:
I create profile without user id first. Next, I create user with profile id. Finally, I update profile with user id.